examples/websocket-retry/websocket_retry.bal

```diff
@@ -53,11 +54,9 @@ service RetryProxyService on new http:Listener(9090) {
     }
 
     //This resource gets invoked upon receiving a new text frame from a client.
-    resource function onText(http:WebSocketCaller caller, string text,
-    boolean finalFrame) {
+    resource function onText(http:WebSocketCaller caller, string text, boolean finalFrame) {
```

Comment:
The column count is > 80 right? Please reduce to 80 or less - see https://github.com/ballerina-platform/ballerina-lang/blob/master/docs/bbe-guidelines/bbe-guidelines.md. Please check everywhere.